src/energyplus/ForwardTranslator/ForwardTranslateScheduleRuleset.cpp

```diff
@@ -148,6 +148,8 @@ boost::optional<IdfObject> ForwardTranslator::translateScheduleRuleset( Schedule
     ScheduleDay defaultDaySchedule = modelObject.defaultDaySchedule();
     ScheduleDay summerDesignDaySchedule = modelObject.summerDesignDaySchedule();
     ScheduleDay winterDesignDaySchedule = modelObject.winterDesignDaySchedule();
+    // ScheduleRuleset is the one carrying the Holiday Schedule
+    ScheduleDay holidayDaySchedule = modelObject.holidaySchedule();
```

Comment:
I say we change "holidayDaySchedule" to "holidaySchedule" for consistency. In the week schedule struct as well.

Reply:
No problem

Reply:
Done